src/SprykerSdk/Zed/AopSdk/Business/OpenApi/Builder/OpenApiCodeBuilder.php

```diff
@@ -0,0 +1,732 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AopSdk\Business\OpenApi\Builder;
+
+use cebe\openapi\Reader;
+use cebe\openapi\spec\OpenApi;
+use cebe\openapi\spec\Operation;
+use cebe\openapi\spec\PathItem;
+use cebe\openapi\spec\Reference;
+use cebe\openapi\spec\Schema;
+use Doctrine\Inflector\Inflector;
+use Generated\Shared\Transfer\MessageTransfer;
+use Generated\Shared\Transfer\OpenApiRequestTransfer;
+use Generated\Shared\Transfer\OpenApiResponseTransfer;
+use SprykerSdk\Zed\AopSdk\AopSdkConfig;
+use Symfony\Component\Process\Process;
+
+class OpenApiCodeBuilder implements OpenApiCodeBuilderInterface
+{
+    /**
+     * @var string
+     */
+    protected const SPRYKER = 'Spryker';
+
+    /**
+     * @var string
+     */
+    protected const CORE = 'core';
+
+    /**
+     * @var \SprykerSdk\Zed\AopSdk\AopSdkConfig
+     */
+    protected AopSdkConfig $config;
+
+    /**
+     * @var \Generated\Shared\Transfer\OpenApiResponseTransfer
+     */
+    protected OpenApiResponseTransfer $openApiResponseTransfer;
+
+    /**
+     * @var \Doctrine\Inflector\Inflector
+     */
+    protected Inflector $inflector;
+
+    /**
+     * @var string
+     */
+    protected string $sprykMode = 'project';
+
+    /**
+     * @var array
+     */
+    protected array $parseProperties = [];
```

Comment:
This is still not expressive and might not even be needed when you pass around things like this.